frontend/test/metabase/scenarios/embedding/embedding-full-app.cy.spec.js

```diff
@@ -11,49 +11,43 @@ describe("scenarios > embedding > full app", () => {
   });
 
   describe("navigation", () => {
-    it("should show the top nav and breadcrumbs by default", () => {
+    it("should hide the top nav by default", () => {
       visitUrl({ url: "/" });
       cy.findByText("Our analytics").should("be.visible");
-      cy.findByTestId("main-logo").should("be.visible");
-    });
-
-    it("should hide the top nav by a param", () => {
-      visitUrl({ url: "/", qs: { top_nav: false } });
-      cy.findByText("Our analytics").should("not.exist");
       cy.findByTestId("main-logo").should("not.exist");
     });
 
-    it("should hide the top nav when all nav elements are hidden", () => {
-      visitUrl({ url: "/", qs: { breadcrumbs: false } });
-      cy.findByText("Our analytics").should("not.exist");
-      cy.findByTestId("main-logo").should("not.exist");
+    it("should show breadcrumbs by default when top nav is enabled", () => {
```

Comment:
I revert the whole test whole to the version before this PR https://github.com/metabase/metabase/pull/26977/files#diff-20906ed2926160122c9587f8bc9545074e041b76c3a3879fb2f0f07847fc2821

But keep this new test to ensure the behavior of breadcrumbs is correct.